packages/shorebird_cli/test/src/command_runner_test.dart

```diff
@@ -274,6 +271,35 @@ Run ${lightCyan.wrap('shorebird upgrade')} to upgrade.'''),
       });
     });
 
+    group('on command failure', () {
+      group('when running with --verbose', () {
+        setUp(() {
+          when(() => logger.level).thenReturn(Level.verbose);
+        });
+
+        test('does not suggest running with --verbose', () async {
+          // This will fail due to missing scoped overrides.
```

Comment:
What do you mean by this comment?

Reply:
Improved this. The release android command being invoked below will fail because `runWithOverrides` does not provide the scoped dependencies required.